ana/toa_vref/toa_efficiency.py

```diff
@@ -0,0 +1,40 @@
+# Calculates TOA_EFFICIENCY for a given dataset,
+# requires a "toa_vref_scan.csv" file to run correctly.
+# Then, plots the efficiency versus toa_vref and saves the plot.
+
+import pandas as pd
+import matplotlib.pyplot as plt
+
+fp = 'toa_vref_scan.csv' # your file path here
+
+data = pd.read_csv(fp,
+                   skiprows = 1)
```

Comment:
See https://github.com/LDMX-Software/pflib/blob/7e9cec455051969671d18223bd8db990c1266228/ana/trim_inv/trim_inv_align.py#L9-L12

You can add a few lines of code before attempting to `from read import read_pflib_csv` and then you have access to the `read_pflib_csv` function which can do this `skiprows` for you and parse in the run parameters stored in the JSON header to (for example) include in the plot legend or title.